The Getting Started Download has been moved

jgold6 opened the pull request:
It is no longer in the Mobile Section. It i snow in the Deployment section of the settings blade. Also it no longer says "Getting Started" it now says "Quickstart".

azurecla commented:
Hi __@get2chhavi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@get2chhavi 
Thanks, Chhavi.

cjgronlund commented:
No CLA is necesary, BTW.

jgold6 committed:
The Getting Started Download has been moved

It is no longer in the Mobile Section. It i snow in the Deployment section of the settings blade. Also it no longer says "Getting Started" it now says "Quickstart".
